src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java

```diff
@@ -1375,7 +1383,8 @@ public AuditLogEntry createAuditLogEntry(GuildImpl guild, JSONObject entryJson,
         final JSONObject options = entryJson.isNull("options") ? null : entryJson.getJSONObject("options");
         final String reason = entryJson.optString("reason", null);
 
-        final UserImpl user = (UserImpl) createFakeUser(userJson, false);
+        final UserImpl user = userJson == null ? null : (UserImpl) createFakeUser(userJson, false);
+        final WebhookImpl webhook = webhookJson == null ? null : (WebhookImpl) createWebhook(webhookJson);
```

Comment:
since these two methods always return the specific implementation it would make more sense to just change the return type rather than doing a typecast